rentable.rb

```diff
@@ -0,0 +1,20 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  @rentals = []
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+
+    person.rentals << self
+    book.rentals << self
+
+    self.class.all << self
+  end
+
+  def self.all
+    @rentals
+  end
```

Comment:
- [ ] Kindly also explain the use of this implementation in the next submission. I don't really think it is useful and would advise you to remove it.

Reply:
done